sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2021-2022.bpmn

```diff
@@ -204,1796 +220,1975 @@ dmnX (comes from dmn)</bpmn:text>
+          <zeebe:output source="=calculatedDataStudentAddtionalTransportation.calculatedDataAdditionalTransportCost" target="calculatedDataAdditionalTransportCost" />
+          <zeebe:output source="=calculatedDataStudentAddtionalTransportation.calculatedDataAdditionalTransportWeeks" target="calculatedDataAdditionalTransportWeeks" />
+          <zeebe:output source="=calculatedDataStudentAddtionalTransportation.calculatedDataAdditionalTransportPlacement" target="calculatedDataAdditionalTransportPlacement" />
+        </zeebe:ioMapping>
+        <zeebe:script expression="={&#10;  calculatedDataAdditionalTransportKm: 0 ,&#10;  calculatedDataAdditionalTransportCost: 0,&#10;  calculatedDataAdditionalTransportWeeks: 0,&#10;  calculatedDataAdditionalTransportPlacement: false&#10;}" resultVariable="calculatedDataStudentAddtionalTransportation" />
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_14w7mr0</bpmn:incoming>
-      <bpmn:outgoing>Flow_16zqfik</bpmn:outgoing>
-    </bpmn:businessRuleTask>
-    <bpmn:businessRuleTask id="Activity_0qzkp53" name="dmnPartTimeAwardAllowableLimits">
+      <bpmn:incoming>Flow_1mtcoea</bpmn:incoming>
+      <bpmn:outgoing>Flow_1me1lzd</bpmn:outgoing>
+    </bpmn:scriptTask>
+    <bpmn:scriptTask id="Activity_1q1wdvs" name="Additional Transportation">
       <bpmn:extensionElements>
-        <zeebe:calledDecision decisionId="dmnPartTimeAwardAllowableLimits" resultVariable="dmnPartTimeAwardAllowableLimits" />
+        <zeebe:ioMapping>
+          <zeebe:output source="=calculatedDataStudentAddtionalTransportation.calculatedDataAdditionalTransportKm" target="calculatedDataAdditionalTransportKm" />
+          <zeebe:output source="=calculatedDataStudentAddtionalTransportation.calculatedDataAdditionalTransportCost" target="calculatedDataAdditionalTransportCost" />
+          <zeebe:output source="=calculatedDataStudentAddtionalTransportation.calculatedDataAdditionalTransportWeeks" target="calculatedDataAdditionalTransportWeeks" />
+          <zeebe:output source="=calculatedDataStudentAddtionalTransportation.calculatedDataAdditionalTransportPlacement" target="calculatedDataAdditionalTransportPlacement" />
+        </zeebe:ioMapping>
+        <zeebe:script expression="={&#10;  calculatedDataAdditionalTransportKm: (if calculatedDataAdditionalTransportKm = null then 0 else calculatedDataAdditionalTransportKm),&#10;  calculatedDataAdditionalTransportCost: (if calculatedDataAdditionalTransportCost = null then 0 else calculatedDataAdditionalTransportCost),&#10;  calculatedDataAdditionalTransportWeeks: (if calculatedDataAdditionalTransportWeeks = null then 0 else calculatedDataAdditionalTransportWeeks),&#10;  calculatedDataAdditionalTransportPlacement: (calculatedDataAdditionalTransportPlacement = &#34;yes&#34;)&#10;} &#10;  " resultVariable="calculatedDataStudentAddtionalTransportation" />
```

Comment:
If condition is not required here when the answer is "yes"

![image](https://github.com/bcgov/SIMS/assets/54600590/c2151aa3-eb74-4815-9cc1-9c1872e160f5)

Reply:
Yes, you're right. Changed.